src/02-data-types.js

```diff
@@ -127,7 +158,23 @@ function isOdd(num) {}
  *  isTruthy(null);
  *  //> false
  */
-function isTruthy(val) {}
+function isTruthy(val) {
+  switch (val) {
+    case true:
+      return true;
+    case null:
+      return false;
+    case undefined:
+      return false;
+    case "":
+      return false;
+    case 0:
+        return false;
+    case false:
+        return false;
+  }
+  return Boolean(val);
```

Comment:
While this works, this is a very verbose way to solve this.  Look up some shorter ways to evaluate how to determine truthiness in JS.